docs/howto/install_package.md

```diff
@@ -0,0 +1,103 @@
+# HOWTO: Install package
+
+## Introduction
+A package can be installed using `elastic-package install` command. This command uses the Kibana API to install the package in Kibana.
+
+## Kibana < 8.7.0
+For versions of `Kibana<8.7.0`, the packages must be exposed via the Package Registry.
+
+In case of development, this means that the package should be built previously and then the Elastic stack must be started. Or at least, package-registry service needs to be restarted in the Elastic stack.
+
+```shell
+eval "$(elastic-package stack shellinit)"
+elastic-package build -v
+elastic-package stack up -v -d  # elastic-package stack up -v -d --services package-registry
+elastic-package install -v
+```
+
+## Kibana >= 8.7.0
+Starting with Kibana 8.7.0, there is a new API to install the packages directly without the need to be exposed in Package Registry. It can be use zip files built using `elastic-package build`command.
+
+**NOTE**: These methods assume that the packages have been validated previously.
```

Comment:
@jsoriano Ideally `install` would run the validate in advance to ensure `elastic-package` never tries to upload an invalid package to Kibana. At the same time I see if the package is already in a zip file, elastic-package would have to unpackage it?

The more general goal on my end is that from a user perspective, it just works and they know about invalidate packages etc. as early as possible.

Reply:
I don't have a strong opinion here so I will leave the decision to @mrodm, but it looks to me like mixing responsibilities. Also, the current implementation based on the registry doesn't perform any validation.

The installer itself (Kibana) will also perform validation steps. Explicit like the signature validation, or implicit like rejecting packages with invalid JSONs. But also shouldn't care about strict linting.

We also have to take into account that we have use cases for legit packages that wouldn't pass strict validation:
* Some packages built from repositories out of integrations repository disable validations or may have old versions of elastic-package.
* Old packages may not comply with most recent practices.
* Packages may intentionally include features not fully supported yet, for testing/development purposes.
* Packages including latest features won't validate if the user is not using a recent enough version of elastic-package.

Reply:
> Old packages may not comply with most recent practices.

This triggered a thought on my end: Should the upload API be restricted to only support most recent package spec version?

Reply:
> > Old packages may not comply with most recent practices.
> 
> This triggered a thought on my end: Should the upload API be restricted to only support most recent package spec version?

Something like this would be aligned with an [internal] discussion we are having about basing the compatibility of packages in Kibana to be based on the spec version, instead of on the Kibana version (https://github.com/elastic/ingest-dev/issues/1622#issuecomment-1451776315).

Kibana would be restricted to support a range of spec versions if that happens.

Reply:
@ruflin I would prefer that `elastic-package install` command would not do validation in packages, just the installation as @jsoriano suggested. So it keeps doing as in the current implementation.

Following that, I've created this PR https://github.com/elastic/elastic-package/pull/1176 to skip validation that I missed here leaving it as `false`.

Reply:
I think a dev should be able to skip validation if they want, but I'm concerned if we skip it by default. I assume we have different users in mind. The user I'm targeting has not experience with elastic-package and only wants to install package as quickly as possible. If there are errors, this user is not able to debug it in Kibana.

I compare it to a github repo that has a commit hook. It can be skipped, but by default it is always happening.

Reply:
I guess we could re-use the parameter `--skip-validation` from the elastic-package build command and use it also here.

So the commands would be something like :
```
# running validation of the package
elastic-package install --zip path -v
elastic-package install -v

# running without validation of the package
elastic-package install --zip path -v --skip-validation
elastic-package install -v --skip-validation
```

WDYT @ruflin @jsoriano ? I could update the PR #1176 with these changes if we agree.

Reply:
👍